src/WebRTCSession.cpp

```diff
@@ -1043,13 +1097,32 @@ WebRTCSession::toggleMicMute()
 void
 WebRTCSession::toggleCameraView()
 {
-        if (insetSinkPad_) {
+        if (localPiPSinkPad_) {
                 guint zorder;
-                g_object_get(insetSinkPad_, "zorder", &zorder, nullptr);
-                g_object_set(insetSinkPad_, "zorder", zorder ? 0 : 2, nullptr);
+                g_object_get(localPiPSinkPad_, "zorder", &zorder, nullptr);
+                g_object_set(localPiPSinkPad_, "zorder", zorder ? 0 : 2, nullptr);
```

Comment:
Hm, why does the zorder need to be 0 or 2? To be below or in front of 1?

Reply:
Yep, GStreamer sets the zorder to 2 by default when inserting the PiP (although I set it explicitly in the code anyway). The surrounding video sits at 1.

Reply:
And you do set it to 0 to hide it then? That makes sense. Thanks for the explanation!